programs/cleos/main.cpp

```diff
@@ -209,6 +227,8 @@ void add_standard_transaction_options(CLI::App* cmd, string default_permission =
    cmd->add_flag("--return-packed", tx_return_packed, localized("Used in conjunction with --dont-broadcast to get the packed transaction"));
    cmd->add_option("-r,--ref-block", tx_ref_block_num_or_id, (localized("Set the reference block num or block id used for TAPOS (Transaction as Proof-of-Stake)")));
    cmd->add_flag("--use-old-rpc", tx_use_old_rpc, localized("Use old RPC push_transaction, rather than new RPC send_transaction"));
+   cmd->add_option("--compression", tx_compression, localized("Compression for transaction 'none' or 'zlib'"))->transform(
```

Comment:
Maybe also list the defaults somewhere.

Reply:
```
  --compression ENUM:value in {none->0,zlib->1} OR {0,1}
                              Compression for transaction 'none' or 'zlib'
```

Reply:
Not really sure I like the output of CLI11 for this, but I guess it is fine.